### DISCUSSION TITLE
More convenient customization

### DISCUSSION THREAD

**Discussion by abitwhy** [LeaVerou/nudeui#10] [Ideas]
You can use the class selector to make customization more convenient.

```diff
- <input type="checkbox" class="nd-switch" style="font-size: 200%">
+ <input type="checkbox" class="big nd-switch">
```
And modifying styles directly in CSS as the final way to customize.

```css
.nd-switch.big {
  font-size: 200%;
}
```

By the way, it is recommended that witting styles in CSS to fitting the concept of separation of concern.

**Comment by LeaVerou** [comment]
I’m well aware of separation of concerns, but this is a demo, and there's value in having all the pieces right there in the demo code.
Also, `.big` is a presentational class name, and thus, ideally avoided.

**Reply by abitwhy** [reply]
Yeh, the current demo is simple and complete. It was just a suggestion of using the built-in styles by the class selector to improve the user experience. Of course, you need to introduce your build-in styles to users first and tell the detail later for those who want to customize. And I see some customization is not easy for users. For example, [`Switch`](https://nudeui.com/nd-switch/) with larger/smaller thumb demos use CSS variables to control the margins, it’s clever but may be confusing for some users.

Besides, `.big` is only for temporary expression. When you build your built-in styles, you‘d better consider some systematic class selectors.
